assets/src/blocks/Submenu/makeHierarchical.js

```diff
@@ -0,0 +1,33 @@
+export const makeHierarchical = headings => {
+  let previousMenuItem;
+
+  return headings.reduce((menuItems, heading) => {
+    const { level, shouldLink, anchor, content, style } = heading;
+
+    // const parent = deeperThanPrevious ? previousHeading.children : menuItems;
+    let possibleParent = previousMenuItem || menuItems;
+
+    while (possibleParent.level && possibleParent.level >= level) {
+      possibleParent = possibleParent.parent;
+    }
+
+    const parent = possibleParent;
+
+    const container = parent === menuItems ? menuItems : parent.children;
+
+    const menuItem = {
+      text: content,
+      style: style,
```

Comment:
Can just write `style` here 🙂

Reply:
Yeah that's one of the parts I still need to clean up :+1: